calculator.rb

```diff
@@ -0,0 +1,102 @@
+# Calculator Program
+
+# stores valid input options for math operations
+valid_maths = [
+  "add",
+  "+",
+  "subtract",
+  "-",
+  "multiply",
+  "*",
+  "divide",
+  "/",
+  "exponent",
+  "^",
+  "modulo",
+  "%"
+]
+
+# method for text displaying available math options
+def maths_text
+  puts "  Add (+)"
+  puts "  Subtract (-)"
+  puts "  Multiply (*)"
+  puts "  Divide (/)"
+  puts "  Exponent (^)"
+  puts "  Modulo (%)"
+end  
```

Comment:
I love that this display functionality is pulled into a method! Keep doing things like this! Also, I like my method names to usually be actions or action-like. Consider maybe changing this method name to `display_maths_text` or `show_maths_text`?